src/rules/tags-check.js

```diff
@@ -27,7 +27,7 @@ var tagsTypings = {
     }
 };
 
-function assign(target) {
+var assign = function(target) {
```

Comment:
![image](https://pics.me.me/let-var-const-what-the-hell-is-this-a-js-64473395.png)